Quality.md

```diff
@@ -0,0 +1,31 @@
+## Quality Assistance
+
+# Quality Coach
+
+> I work with Octopus team members to coach them to deliver better overall quality.
+
+### I am a coach
+
+- I coach teams to deliver a product that is user-friendly, efficient, consistent, robust, and useful to customers.
+- I can manage difficult conversations and tactfully challenge others, leaving them with a positive impression of myself and Octopus.
+- I can effectively coach people and pass on my knowledge.
+- I help teams to achieve better business value.
+- I recognize and accept that change is constant, and my approach allows me to tactfully challenge, or confidently adapt, depending on which I deem appropriate.
+- I possess writing skills that let me be clear, concise, articulate, and persuasive in all of my outgoing communications.
+- I know how to make pragmatic decisions in order to maintain influence.
+
+### I ensure Octopus teams can ship with confidence
+
+- I am an advocate for quality.
+- I help engineers refine requirements and identify the right level of testing
```

Comment:
```suggestion
- I help engineers refine requirements and identify the optimal kind, level, and timing for testing.
```

Reply:
I help engineers refine requirements, prioritize tasks, identify the optimal kind, level, and timing for testing.

Reply:
> prioritize tasks

I disagree that that is part of the role